app/shared_features/camera/VideoRecorder.js

```diff
@@ -180,6 +204,13 @@ const styles = StyleSheet.create({
         height: 30,
         paddingTop: 5,
         textAlign: 'center'
+    },
+    switchCamera: {
+        backgroundColor: 'transparent',
+        marginRight: 40,
+        flexDirection: 'row',
+        justifyContent: 'flex-end',
+        alignItems: 'flex-start',
     }
 });
 
```

Comment:
Switch camera mid record breaks, should not be able to switch camera once it's started